pkg/littlefs/Makefile

```diff
@@ -1,6 +1,7 @@
 PKG_NAME=littlefs
 PKG_URL=https://github.com/geky/littlefs.git
-PKG_VERSION=v1.3
+# v1.6.1
+PKG_VERSION=cb62bf2188854c5b7c44383571ebb19a414e6137
```

Comment:
Why not use the `v1.6.1` (or `v1.6.2`) tag instead of the full commit hash?

Reply:
I saw some debate on another PR (which I can't remember) about this and I think the decision was to go for the commit hash.

Reply:
Found [the discussion](https://github.com/RIOT-OS/RIOT/issues/9966). Never mind me, then :)